pages/mission.vue

```diff
@@ -0,0 +1,249 @@
+              <p>{{ mission.livrable }}</p>
+            </div>
+            <div id="atecnaPlus">
+              <h4>Les + Atecna</h4>
+              <p>{{ mission.atecnaPlus }}</p>
+            </div>
+          </b-col>
+        </b-row>
+      </b-container>
+    </div>
+  </div>
+</template>
+
+<script>
+  const utils = require('../store/utils')
+
+  export default {
+    data() {
+      return {
+        mission: {},
+        logoCart: this.$cartIcon,
+        logoCalendar: this.$calendarIcon,
+        logoUser: this.$userIcon,
+        logoMapMarker: this.$mapMarkerIcon,
+        logoAtecna: this.$atecnaLogo
+      }
+    },
+    mounted() {
+
+      console.log('params = ', this.$route.query.id)
+      const params = {
+        id: this.$route.query.id,
+        token: this.$auth.getToken('local')
+      }
+
+      this.$axios.get('/api/mission/getOne', { params })
+        .then(response => {
+          this.mission = response.data.mission
+        })
+    },
+    methods: {
+      edit() {
+
+      },
+      imgUrlAlt(event) {
+        event.target.src = this.$imageFallback
+      },
+      downloadWithCSS() {
+        utils.makePDF(this.mission)
+      }
+    }
+  }
+</script>
+
+<style>
+    #page {
+        display: flex;
+    }
+    #left--part {
+        width: 30%;
+        background-color: #f1c40f;
+        height: 100vh;
+    }
+    #image img {
+        width: 100%;
+        object-fit: cover;
+        height: 10em;
+    }
+    #buttons--mission {
+        display: flex;
+        justify-content: center;
+        width: 100%;
+    }
+    button {
+        margin: 1em 0.3em;
+        background-color: #2f3640;
+        color: white;
+        border-radius: 7px;
+        border: none;
+    }
+    #picto {
+        width: 100%;
+        display: flex;
+        justify-content: center;
+        flex-flow: wrap;
+        text-align: center;
+    }
+    .round--picto {
+        width: 50px;
+        height: 50px;
+        border-radius: 50%;
+        background-color: #f5f6fa;
+        color: black;
+        margin: 1em;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+    #environnement {
+        display: flex;
+        justify-content: center;
+    }
+    #env {
+        display: flex;
+        flex-flow: wrap;
+        justify-content: center;
+    }
+    #picto--env {
+        display: flex;
+        flex-direction: column !important;
+        justify-content: center;
+        /* flex-flow: wrap; */
+    }
+    #picto--env p {
+        text-align: center;
+    }
+    #right--part {
+        width: 70%;
+        height: 100vh;
+    }
+    #logos {
+        width: 100%;
+        display: flex;
+        justify-content: space-between;
+        padding: 1em;
+    }
+    #atecna {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+    #atecna img{
+        width: 80%;
+    }
+    #logos #client--logo img {
+        width: 10%;
```

Comment:
can be dangerous if there is no min height & width for keeping an aspect ratio with width percentage, img might get stretch horizontally